modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-http-interface/httpInterfacesWebClientConfiguration.mustache

```diff
@@ -28,7 +28,7 @@ public abstract class HttpInterfacesAbstractConfigurator {
 {{#apis}}
     @Bean(name = "{{configPackage}}.HttpInterfacesAbstractConfigurator.{{classVarName}}")
     {{classname}} {{classVarName}}HttpProxy() {
-        HttpServiceProxyFactory factory = HttpServiceProxyFactory.builder(WebClientAdapter.forClient(webClient)).build();
+        HttpServiceProxyFactory factory = HttpServiceProxyFactory.builderFor(WebClientAdapter.forClient(webClient)).build();
```

Comment:
This is marked as deprecated, replacing it with `builderFor`, which is also in line with the rest of the `httpInterfaces*Configuration.mustache` templates